examples/indexer/java/AccountsAppID.java

```diff
@@ -0,0 +1,40 @@
+// AccountsAppID.java
+// requires java-algorand-sdk 1.4.0 or higher (see pom.xml)
+package com.algorand.javatest.indexer;
+
+import com.algorand.algosdk.v2.client.common.IndexerClient;
+
+import com.algorand.algosdk.v2.client.model.AccountsResponse;
+import com.algorand.algosdk.v2.client.common.Response;
+import com.algorand.algosdk.v2.client.common.Client;
+
+import org.json.JSONObject;
+
+public class AccountsAppID {
+    public Client indexerInstance = null;
+    // utility function to connect to a node
+    private Client connectToNetwork() {
+        
+        // TODO  remove    
+        // final String INDEXER_API_ADDR = "https://indexer-internal-betanet.aws.algodev.network/";
+        // final String INDEXER_TOKEN = "YddOUGbAjHLr1uPZtZwHOvMDmXvR1Zvw1f3Roj2PT1ufenXbNyIxIz0IeznrLbDsF";
+        // final int INDEXER_API_PORT = 443;
+
+        final String INDEXER_API_ADDR = "http://localhost";
+        final String INDEXER_TOKEN = "";
+        final int INDEXER_API_PORT = 59998;
```

Comment:
Should be 8980 throughout 

Reference default setting:
https://github.com/algorand/indexer/blob/7e7f1c34d6f930a424a6030750fe1d7a54006d2d/cmd/algorand-indexer/daemon.go#L115